README.md

```diff
@@ -162,47 +199,6 @@ loadIcons('icons/spectrum-icons.svg');
 
 You can then use the icons in your app. Visit the [Spectrum CSS workflow icon list](http://opensource.adobe.com/spectrum-css/icons/) and click on any icon to get the markup.
 
-### Swapping out icons sets for scaling
```

Comment:
This just isn't relevant anymore, since we advise people to use `spectrum-css-icons.svg`, which has both.